src/api/objects/server/serverCreateObject.lua

```diff
@@ -7,6 +7,12 @@
 ---@return void
 ---@public
 function API_Objects:serverCreateObject(props, x, y, z, h)
+    assert(type(props) == "string" or type(props) == "number")
+
+    if type(props) == "string" then
+        props = GetHashKey(props)
+    end
+
     local object = CreateObject(props, x, y, z - 1, true)
```

Comment:
Why z - 1 ?

Reply:
Idk, it wasn't done by me. I'll delete it